src/overlay/OverlayManagerImpl.cpp

```diff
@@ -373,17 +375,27 @@ OverlayManagerImpl::getFlowControlBytesConfig() const
                   INITIAL_FLOW_CONTROL_SEND_MORE_BATCH_SIZE_BYTES >=
               maxTxSize))
         {
-            return {static_cast<uint32_t>(maxTxSize) +
-                        INITIAL_FLOW_CONTROL_SEND_MORE_BATCH_SIZE_BYTES,
-                    INITIAL_FLOW_CONTROL_SEND_MORE_BATCH_SIZE_BYTES};
+            static_cast<uint32_t>(maxTxSize) +
```

Comment:
This result just gets thrown away. Should it be returned? Also, `maxTxSize` is already a `uint32_t`, so the `static_cast` is unnecessary.

Reply:
oh gosh, great catch! Looks like this was accidentally introduced in the last cleanup commit (so after perf evaluation was done). Tests didn't catch it since they always override these configs. I'll add a test case that uses default configs.